README.md

```diff
@@ -2,51 +2,126 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/tarantool/test-run/badge.svg)](https://coveralls.io/github/tarantool/test-run)
 
-### Test Suite
-
-Bunch of tests, that lay down in the subfolder (recursively) with `suite.ini`
-file. `suite.ini` is basic ini-file, that consists of one section `default`,
-and a number of fields:
-
-* `core`
-* `description` - Test Suite description
-* `script` - shebang file to start tarantool with
-* disables:
-    * `disabled` - tests that must be skipped
-    * `release_disabled` - tests that must be skipped when Tarantool has been
-      builded with `Release`
-    * `valgrind_disabled` - tests that must be skipped when Valgrind is enabled
-* `lua_libs` - paths for lua files, that should be copied into the folder,
-  where server is started (delimited with the space, e.g. `lua_libs=lua/1.lua
-  lua/2.lua`)
-* `long_run` - mark tests as long, enabled only with `--long` option (delimited
-  with the space, e.g. `long_run=t1.test.lua t2.test.lua`)
-* `config` - test configuration file name
-
-Field `core` must be one of:
-
-* `tarantool` - Test-Suite for Functional Testing
-* `app` - Another functional Test-Suite
-* `unittest` - Unit-Testing Test Suite
-
-### Test
-
-Each test consists of files `*.test(.lua|.sql|.py)?`, `*.result`, and may have
-skip condition file `*.skipcond`.  On first run (without `.result`) `.result`
-is generated from output.  Each run, in the beggining, `.skipcond` file is
-executed. In the local env there's object `self`, that's `Test` object. If test
-must be skipped - you must put `self.skip = 1` in this file. Next,
-`.test(.lua|.py)?` is executed and file `.reject` is created, then `.reject` is
-compared with `.result`. If something differs, then 15 last string of this diff
-file are printed and `.reject` file is saving in the `<vardir>/rejects/<suite>`
-subfolder given in options or set localy as `var/rejects/<suite>` by default.
-If not, then `.reject` file is deleted.
+## Test suite configuration
+
+Test suite is a bunch of tests located in a subdirectory (recursively) with a `suite.ini`
+file. The `suite.ini` is a basic ini-file with one section `[default]`.
+
+```ini
+[default]
+core = luatest
+description = Example test suite using luatest
+...
+```
+
+There's a syntax for single- and multiline lists. (TODO see ./sql-tap/suite.ini)
+For example:
+
+```ini
+single_line = first.test.lua second.test.lua third.test.lua
+multiline = first.test.lua ;
+            second.test.lua ;
+            third.test.lua ;
+            ...
+```
+
+Below is a list of configuration values (fields) in the `suite.ini` file.
+
+### General configuration
+
+* `core` — major testing dependency or method
+  Should have one of the following values:
+
+  * `tarantool` — test suite used for functional testing
+  * `app` — TAP-tests, another functional test suite
+  * `unittest` — unit testing test suite
+  * `luatest` — test suite using luatest library
```

Comment:
If you're interested I have a description of all test types [here](https://github.com/tarantool/luatest/issues/191) (under the cut). It reveals one or two main characteristics of a test type (taking care to real usage and informal terminology we have). However it is balanced to be as concise as possible. As for me, current description gives no useful information (sorry).